src/core/renderer.service.ts

```diff
@@ -9,35 +10,30 @@ export interface RendererTester {
 export class RendererService {
   private renderers: Array<{tester: RendererTester, renderer: string}> = [];
   registerRenderer(tester: RendererTester, renderer: string): void {
-    this.renderers.push({tester: tester, renderer: renderer});
+    this.renderers.push({tester, renderer});
   }
   unregisterRenderer(tester: RendererTester, renderer: string): void {
-    const index = this.renderers.indexOf({tester: tester, renderer: renderer});
```

Comment:
This `indexOf` call always returned -1 since we are only able to compare testers (which are essentially functions) via strict equality (i.e. references). `splice` 1st argument may also be negative, see [here](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/splice).